### DISCUSSION TITLE
Old Sidebar replacement

### DISCUSSION THREAD

**Discussion by fevers** [AprilSylph/XKit-Rewritten#1146] [Suggestion Box]
In the previous version of XKit I had the Old Sidebar third-party extension, but it broke sometime after Tumblr did the redesign. Is this a functionality you could consider implementing for XKit Rewritten?
Thank you!

**Comment by marcustyphoon** [comment]
I did try it a while ago, in the sense that I tried directly transporting the contents of Tumblr's menu dropdown into the sidebar, and unfortunately Javascript Don't Work Like That :D It would probably have to be Tumblr implementing something at least somewhat closer to a sidebar menu for these UI items that we could tweak. Could be kind of neat.

**Comment by marcustyphoon** [comment]
Not sure if you have or have seen screenshots of the experimental layout that's being tested on a random subset of users right now, but its navigation sure is basically this! (But on the left, so that aspect's different.)
